include/camp/helpers.hpp

```diff
@@ -79,6 +79,37 @@ namespace type
     using add = T&&;
   }  // end namespace rvref
 
+  namespace ptr
+  {
+    template <class T>
+    struct rem_s {
+      using type = T;
+    };
+    template <class T>
+    struct rem_s<T*> {
+      using type = T;
+    };
+    template <class T>
+    struct rem_s<T**> {
+      using type = T;
+    };
+
+    /// remove pointer from T
+    template <class T>
+    using rem = typename rem_s<T>::type;
+
+    /// add remove pointer to T
+    template <class T>
+    using add = T*;
+  }  // end namespace ptr
+
+  namespace lvptr
```

Comment:
This whole namespace block and contents should come out too, the rvref is because you need two refs added to get an rvalue reference, if you want to have one for working directly with double pointers, I'd actually be fine with that, but I'd rename it to make that clear and have both add and rem variants for it.

Reply:
Makes sense, I removed this namespace to keep it consistent.